### ISSUE TITLE
Hosted engine deployment fails with ascii encoding error

### ISSUE THREAD

**Issue by JacobHenner** [oVirt/ovirt-ansible#147]
```
[ ERROR ]  [WARNING]: Failure using method (v2_runner_on_failed) in callback plugin
         
[ ERROR ] (<ansible.plugins.callback.1_otopi_json.CallbackModule object at 0x1d69f90>):
         
[ ERROR ] 'ascii' codec can't encode character u'\u2018' in position 484: ordinal not in
         
[ ERROR ] range(128)
         
[ ERROR ] Failed to execute stage 'Closing up': Failed executing ansible-playbook

```

On CentOS 7.

No '\u2018' character was entered during the deployment process.

**Comment by machacekondra** [comment]
cc @tiraboschi

**Comment by tiraboschi** [comment]
We had an issue parsing unicode error messages in the logging callback but it's now fixed in ovirt-hosted-engine-setup-2.2.12 as for https://bugzilla.redhat.com/1533500

That one fixes the handling of the error message with unicode chars but not the source cause of it.
I fear it comes from dd due to a permission error on the NFS share, @JacobHenner could you please check that?

**Comment by JacobHenner** [comment]
Looks like it might have been the cause, but due to some other unrelated issues, I was not able to reproduce.
